### DISCUSSION TITLE
Figure Of Merit value difference between ResNet8 and Conv127

### DISCUSSION THREAD

**Discussion by k-haru** [tbepler/topaz#151] [Q&A] [answered]
Hi topaz team,

I used ResNet8 and Conv127 to see the difference between the models and train them via relion v4b.
The results showed that the FOM values were between -6 ~ 6 for the former and -6 ~ 0 for the latter.
If FOM is log-likelihood, then Conv127's FOM makes sense, but with ResNet8 I am not sure.
Can you give me some advice?

Best regards

**Comment by tbepler** [comment, marked as answer]
These are log-likelihood ratios, so they are log(p(y=1)/(1-p(y=1))). It looks like your conv127 model isn't as confident as your resnet8 model, but the range of values doesn't look unreasonable to me.

**Reply by k-haru** [reply]
I understand well! (and re-checked paper)
Conv127 model's lack of confidence was due to not specifying the proper Downscale factor. It now has enough confidence! 
Thanks for your kind advice.
